src/plots/cartesian/axes.js

```diff
@@ -1399,6 +1424,66 @@ function formatLinear(ax, out, hover, extraPrecision, hideexp) {
     out.text = numFormat(out.x, ax, hideexp, extraPrecision);
 }
 
+function formatAngle(ax, out, hover, extraPrecision, hideexp) {
```

Comment:
Ah nice, here it is 🎉  I guess even though it's currently only used for polar, there *is* something nice about having this in `axes.js` (granted that perhaps all of this should go somewhere else outside cartesian at some point) because I could imagine us wanting to support fractions and/or multiples of π on cartesian axes at some point.

But yeah, previous comments hold: we need a cutoff to switch to decimal, and then perhaps hover can use π too!